WORKSPACE

```diff
@@ -17,7 +17,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_register_toolchains", "go_rules_depen
 go_rules_dependencies()
 
 go_register_toolchains(
-    go_version = "1.11.5",
+    go_version = "1.12.5",
```

Comment:
We should probably also update `rules_go` and `bazel_gazelle` as in https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/WORKSPACE#L21

Reply:
I saw that is part of the other PR, will wait for that one to go in first and then LGTM this one :)